### PR TITLE
Modernise release and bump WP version

### PR DESCRIPTION
**Pull request by the-csaba**

### CODE REVIEW

**Comment by the-csaba** [comment]
@james,

Unfortunately, all the line ending has changed, so the diff looking very confusing on Github. Please commit locally and you can see the changes, which is minimal.

**Comment by the-csaba** [comment]
To the Github action to work, we need not just a tag, but a Github Release. So maybe a couple of new releases will be needed if the config not correct

File: weatherzone.php [hunk 1]
```diff
@@ -1,213 +1,213 @@
...
-
-if(defined('ABSPATH') && defined('WPINC')) {
-	if (!isset($GLOBALS["om4_WeatherZone"])) {
-		$GLOBALS["om4_WeatherZone"] = new OM4_WeatherZone();
-	}
-}
+<?php
+/*
+Plugin Name: WeatherZone Embed
+Plugin URI: https://om4.com.au/plugins/weatherzone/
```
Comment (thejamescollins) [inline weatherzone.php]: Can change this to 
https://om4.io/plugins/weatherzone/

File: weatherzone.php [hunk 2]
```diff
@@ -1,213 +1,213 @@
...
+Description: Allows you to embed WeatherZone.com.au weather buttons on your site. Supports both weather forecast and current weather observations buttons.
+Version: 1.2.8
+Author: OM4
+Author URI: https://om4.com.au/plugins/
```
Comment (thejamescollins) [inline weatherzone.php]: Can change to 
https://om4.io/

File: readme.txt [hunk 3]
```diff
@@ -1,140 +1,143 @@
...
-
-= 1.1 =
-* WordPress 3.1 compatibility.
-
-= 1.0.1 =
-* This version allows the [weatherzone] shortcode to be used in sidebar text widgets
\ No newline at end of file
+=== WeatherZone Embed ===
+Contributors: jamescollins, glenn-om4
+Donate link: https://om4.com.au/wordpress-plugins/#donate
```
Comment (thejamescollins) [inline readme.txt]: This can be removed I think, no-one ever uses a donation form :)

**Review by thejamescollins — CHANGES_REQUESTED** [review]
Looking great @om4csaba, thank you :) :)

I've left a few comments re om4.io URL's.

I think we also need to update the POT file.

Then we should be good to release I think.

Thanks

**Comment by the-csaba** [comment]
> Then we should be good to release I think.

Thanks @om4james , I will merge and start playing with the tagging and release

**Pull request merged**